apps/dash/src/styles/index.css

```
@@ -1,16 +1,13 @@
 @import "tailwindcss";
 
 @source "../";
-
-@plugin "tailwindcss-animate";
-
+@import "tw-animate-css";
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)